packages/realm-server/tests/cards/%F0%9F%98%80.gts

```diff
@@ -0,0 +1,7 @@
+// This is a file that contains url encoded file name to assert
+// that the realm server can handle this type of file
+
+import { CardDef } from 'https://cardstack.com/base/card-api';
+export class Test extends CardDef {
+  static displayName = 'test';
+}
```

Comment:
Before this PR, the presence of this file would have caused the realm server to not start. The fact that server test can even run with this file present means that the fix is working.